control-plane/roles/monitoring/defaults/main/main.yaml

```diff
@@ -0,0 +1,7 @@
+---
+metal_monitoring_namespace: monitoring
+monitoring_ingress_dns: "monitoring.{{ metal_control_plane_ingress_dns }}"
+monitoring_tls: yes
+# monitoring_metal_metrics_exporter_version:
```

Comment:
Can be removed.

Reply:
https://github.com/metal-stack/metal-roles/commit/128d5117656e10f8ad53d5cb1e1a8ee16482a18c